docker/bitcoind/start-bitcoind.sh

```diff
@@ -85,6 +83,11 @@ fi
 # an IP and then connect to that machine
 if [[ "$NETWORK" == "regtest" ]] && [[ -n "$CONNECT_HOST" ]]; then
     if [[ "$CONNECT_HOST" == "host.docker.internal" ]]; then
+        ping -c1 $CONNECT_HOST > /dev/null 2>&1
```

Comment:
I would make a note in each place describing why we are adding an extra ip/host to our host file on all these boxes. Would emphasize that this is only used because we do networking with `host.docker.internal` which is not currently supported for linux boxes

Reply:
Will do. Also, if you think of a clean way to do this that doesn't duplicate so much code, let me know (we do the same thing in 4 places in this repo and 1 place in `lnd-engine` ...) let me know. I haven't thought of a clean way so far.